Source/Tools/FEXLogServer/Main.cpp

```diff
@@ -0,0 +1,276 @@
+#include "Common/Config.h"
+#include "Common/SocketLogging.h"
+
+#include "Tools/CommonGUI/IMGui.h"
+
+#include <map>
+#include <unordered_set>
+#include <unistd.h>
+
+namespace Common {
+  const char *GetCharLevel(uint32_t Level) {
+    const char *CharLevel{nullptr};
+
+    switch (Level) {
+    case LogMan::NONE:
+      CharLevel = "NONE";
```

Comment:
Seems like these stringified category names can be returned directly instead of being assigned to a local

Reply:
Resolved.